src/Packagist/WebBundle/Entity/PackageRepository.php

```diff
@@ -279,7 +279,7 @@ public function getFilteredQueryBuilder(array $filters = array())
             $qb->leftJoin('v.tags', 't');
         }
 
-        $qb->orderBy('p.id', 'DESC');
```

Comment:
the order by id should also be kept, to make the sorting stable (but after the abandoned flag)